### PR TITLE
Update Deps 3: Most deps (minus electron)

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] test desktop
- [x] test mobile

This causes redux-saga to give warnings about not using the new all() effect wrapper, but it seems like some other thing (cancel) is causing this with an empty array. See https://github.com/redux-saga/redux-saga/issues/1000

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits e3ea9a741c01...c9512955506d introduce visual changes on linux. <details><summary>:mag_right: 8 changed</summary>

 * Changed: **Profile-Your_Profile_Proof_Menu** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Proof_Menu-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Proof_Menu-c9512955506d.png)  
   Profile-Your_Profile_Proof_Menu comparison
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_HN** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Pending_Proof_Menu_HN-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Pending_Proof_Menu_HN-c9512955506d.png)  
   Profile-Your_Profile_Pending_Proof_Menu_HN comparison
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_DNS** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-c9512955506d.png)  
   Profile-Your_Profile_Pending_Proof_Menu_DNS comparison
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Pending_Proof_Menu-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Pending_Proof_Menu-c9512955506d.png)  
   Profile-Your_Profile_Pending_Proof_Menu comparison
 * Changed: **Profile-Your_Profile_Broken_Proof_Menu** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Broken_Proof_Menu-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Broken_Proof_Menu-c9512955506d.png)  
   Profile-Your_Profile_Broken_Proof_Menu comparison
 * Changed: **Profile-Your_Profile_Broken** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Broken-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Profile-Your_Profile_Broken-c9512955506d.png)  
   Profile-Your_Profile_Broken comparison
 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Markdown-emoji-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Markdown-emoji-c9512955506d.png)  
   Markdown-emoji comparison
 * Changed: **Markdown-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Markdown-Normal-e3ea9a741c01.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Markdown-Normal-c9512955506d.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496163281970-e3ea9a741c01-c9512955506d-linux/Markdown-Normal.png)
</summary>

File: shared/common-adapters/emoji.desktop.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React from 'react'
 import {Emoji} from 'emoji-mart'
 
 // Just the single set we use
-import emojiSet from 'emoji-datasource/sheet_apple_64.png'
+import emojiSet from 'emoji-datasource-apple/img/apple/sheets/64.png'
```
Comment (chrisnojima) [inline shared/common-adapters/emoji.desktop.js]: they broke out specific sheets into their own packages

**Comment by chrisnojima** [comment]
- [ ] visdiff

**Comment by chrisnojima** [comment]
So the visdiff is a timing issue somehow related to the new react. it actually works fine in the app. I think we should leave it and fix it up with the new modal solution we find w/ searchv3

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by keybase-ci-visdiff** [comment]
The commits e04b3eb999ea...a520911b90c7 introduce visual changes on linux. <details><summary>:mag_right: 8 changed</summary>

 * Changed: **Profile-Your_Profile_Proof_Menu** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Proof_Menu-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Proof_Menu-a520911b90c7.png)  
   Profile-Your_Profile_Proof_Menu comparison
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_HN** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Pending_Proof_Menu_HN-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Pending_Proof_Menu_HN-a520911b90c7.png)  
   Profile-Your_Profile_Pending_Proof_Menu_HN comparison
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_DNS** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-a520911b90c7.png)  
   Profile-Your_Profile_Pending_Proof_Menu_DNS comparison
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Pending_Proof_Menu-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Pending_Proof_Menu-a520911b90c7.png)  
   Profile-Your_Profile_Pending_Proof_Menu comparison
 * Changed: **Profile-Your_Profile_Broken_Proof_Menu** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Broken_Proof_Menu-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Broken_Proof_Menu-a520911b90c7.png)  
   Profile-Your_Profile_Broken_Proof_Menu comparison
 * Changed: **Profile-Your_Profile_Broken** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Broken-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Profile-Your_Profile_Broken-a520911b90c7.png)  
   Profile-Your_Profile_Broken comparison
 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Markdown-emoji-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Markdown-emoji-a520911b90c7.png)  
   Markdown-emoji comparison
 * Changed: **Markdown-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Markdown-Normal-e04b3eb999ea.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Markdown-Normal-a520911b90c7.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496264913138-e04b3eb999ea-a520911b90c7-linux/Markdown-Normal.png)
</summary>

**Pull request merged**